docker-compose.yml

```
@@ -0,0 +1,31 @@
+# Copy this into a file called `docker-compose.yml`
+version: "3"
+
+services:
+  dev:
+    image: todo-app:dev
+    env_file:
+      - ./.env
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: development
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./todo_app:/app/todo_app
```

I don't think volume is doing very much at the moment - try changing some code and refreshing the page; do you see the change?

We just need to tweak this to match your chosen directory structure:
```suggestion
      - ./todo_app:/opt/todo_app
```